package.json

```diff
@@ -58,6 +58,7 @@
   },
   "scripts": {
     "dev": "turbo dev --filter=@clerk/* --filter=!@clerk/expo",
+    "dev:current": "turbo dev:current",
```

Comment:
I like this approach :) 
Sorry in advance for being nitpicky, however, I feel that the name of this command is slightly confusing as someone going through the `package.json` file needs to be aware of the `-- --filter` option in `clerk_go/Makefile`. Otherwise, running `dev:current` without arguments will run `build` and `dev` for all packages, is that correct?